commons/commons-serialization/src/test/kotlin/ac/uk/ebi/biostd/test/SubTestFactory.kt

```diff
@@ -22,7 +22,7 @@ fun createVenousBloodMonocyte() = submission("S-IHECRE00000919.1") {
 
     section("Study") {
         accNo = "SECT-001"
-        attribute(name = "Title", value = "venous blood, Monocyte")
+        attribute(name = "Title", value = "venous blood, \n Monocyte")
```

Comment:
I think we should also include this scenario in the [integration test](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/MultipartFileSubmissionApiTest.kt#L68) we have for XLSX